client/src/ts/trainees/InternalProject.tsx

```diff
@@ -0,0 +1,330 @@
+    /*
+    *
+    * Here will be the api call for storing the changed data in the database, currently the api call for the members page is here as a placeholder since the required function is not implemented yet
+    * Check if data is stored correctly in database after api is implemented
+    * 
+    api
+      .patch(`/users/${props.match.params.id}`, data, {
+        headers: { Authorization: `Bearer ${localStorage.getItem("token")}` },
+      })
+      .then((res: any) => {
+        if (res.status === 200) {
+          if (mounted) {
+            showSuccessMessage("Aktualisierung des Profils war erfolgreich!");
+            getInternalProjectDetails();
+          }
+        }
+      })
+      .catch((err: any) => {
+        if (err.response.status === 401) {
+          dispatchAuth({ type: authReducerActionType.deauthenticate });
+        } else if (err.response.status === 500) {
+          showErrorMessage("Aktualisierung ist fehlgeschlagen!");
+        }
+      });
+      */
+
+    // Clean-up function
+    return () => {
+      mounted = false;
+    };
+  };
+
+  /**
+   * Returns the interface
+   */
+  return (
+    <div>
+      <InternalProjectInformationDialog
+        name={name}
+        kuerzel={kuerzel}
+        traineegeneration={traineegeneration}
+        kickoff={kickoff}
+        angebotAbgegeben={angebotAbgegeben}
+        apAbgegeben={apAbgegeben}
+        apDatum={apDatum}
+        zpDatum={zpDatum}
+        projektmitglieder={projektmitglieder}
+        dlAbgegeben={dlAbgegeben}
+        zpAbgegeben={zpAbgegeben}
+        qualitaetsmanager={qualitaetsmanager}
+        internalProjectInfoDialogOpen={internalProjectInfoDialogOpen}
+        handleInternalProjectInfoDialogClose={handleInternalProjectInfoDialogClose}
+        setName={setName}
+        setKuerzel={setKuerzel}
+        setTraineegeneration={setTraineegeneration}
+        setKickoff={setKickoff}
+        setAngebotAbgegeben={setAngebotAbgegeben}
+        setApDatum={setApDatum}
+        setApAbgegeben={setApAbgegeben}
+        setZpDatum={setZpDatum}
+        setZpAbgegeben={setZpAbgegeben}
+        setDlAbgegeben={setDlAbgegeben}
+        setProjektmitglieder={setProjektmitglieder}
+        setQualitaetsmanager={setQualitaetsmanager}
+        updateInternalProjectDetails={updateInternalProjectDetails}
+      />
+
+      <div className="content-page">
+        <Paper className={classes.paper}>
+          <div className={classes.categoryHeader}>
+            <div>
+              <Typography variant="h6">
+                <strong>Informationen zum internen Projekt</strong>
+              </Typography>
+            </div>
+            <div>
+              {doesPermissionsHaveSomeOf(auth.permissions, [15]) ? (
+                <IconButton onClick={(event) => handleInternalProjectInfoDialogOpen(event)}>
+                  <Edit fontSize="inherit" />
+                </IconButton>
+              ) : null}
+            </div>
+          </div>
+          <br></br>
```

Comment:
`</br>` is an invalid html element because br is self closing.
Use `<br>` or `<br/>`.

```suggestion
          <br/>
```